### PR TITLE
Django110

### PR DESCRIPTION
**Pull request by madhulika95b**
@somexpert @ldko 
Upgraded to django 1.10 version and ready for review.

### CODE REVIEW

**Comment by somexpert** [comment]
Changes look good, tests pass, and app seems to be working fine :thumbsup:

File: tox.ini [hunk 1]
```diff
@@ -29,7 +29,6 @@ commands =
 
     # Override pipenv version of Django as needed by test envs. Use pip so that
     # Pipfile.lock doesn't end up changed by this process.
-    django19: pip install Django~=1.9.0
     django110: pip install Django~=1.10.0
```
Comment (somexpert) [inline tox.ini]: We should be able to just remove this line, as for now we're installing the correct version with the Pipfile, so no need to "correct" it with this manual installation. May need to purge your old tox environments when running the tests.

Comment (somexpert) [reply tox.ini]: Also, can we go ahead and improve the indentation of this file? I'm absolutely not a fan of having the variables under an environment start at the same indentation level as the environment name.

Comment (ldko) [reply tox.ini]: If we delete it, could the Pipfile.lock change to a different bug fix release or something which would change the version and also possibly dependency versions?

Comment (madhulika95b) [reply tox.ini]: I don't think any changes to tox.ini would change Pipfile.lock

Comment (ldko) [reply tox.ini]: Oh wait, you are saying the tests will use the lock file anyway, I misunderstood.

**Pull request merged**